### PR TITLE
Create common parent for Jpa*AttributeEntity

### LINKED ISSUE #10071: Create common parent for Jpa*AttributeEntity
Originally it comes from https://github.com/keycloak/keycloak/pull/9997#discussion_r801515317

All attribute entities contains similar code. This issue is about creating common parent for those as proposed in the comment above.

### PR DESCRIPTION
**Pull request by vramik**
Closes #10071

### CODE REVIEW

File: model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/JpaAttributeEntity.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates
```
Comment (ahus1) [inline model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/JpaAttributeEntity.java:2]:
```suggestion
 * Copyright 2022 Red Hat, Inc. and/or its affiliates
```

File: model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/JpaAttributeEntity.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    @Override
+    public int hashCode() {
+        return getClass().hashCode();
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj) return true;
+        if (!(obj instanceof JpaAttributeEntity)) return false;
+        JpaAttributeEntity that = (JpaAttributeEntity) obj;
```
Comment (ahus1) [inline model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/JpaAttributeEntity.java:96]: The following avoids a warning in my IDE about "Raw use of parameterized class 'JpaAttributeEntity' "

```suggestion
        JpaAttributeEntity<?> that = (JpaAttributeEntity<?>) obj;
```

**Review by ahus1 — DISMISSED** [review]
Tested locally, works well. Also reviewed all code. 

Found two nitpicking things in the code. Can be merged regardless of these two comments, therefore approving anyway.

**Review by ahus1 — APPROVED** [review]
LGTM, all comment resolved!

**Review by hmlnarik — APPROVED** [review]
Thank you for the PR! Ready for merging once the checks pass.

**Comment by hmlnarik** [comment]
The only remaining testsuite is _Quarkus tests_ that are irrelevant for this PR. Merging.

**Pull request merged**